mmdet/models/detectors/centernet.py

```diff
@@ -0,0 +1,131 @@
+import torch
+
+from mmdet.core import bbox2result
+from mmdet.models.builder import DETECTORS
+from .cornernet import CornerNet
+
+
+def flip_feature_map(src_tensor, flip_direction):
+    """flip feature map base on flip_direction.
+
+    Args:
+        src_tensor (Tensor): input feature map, shape (B, C, H, W).
+        flip_direction (str): The flipping direction. Options are
+          'horizontal', 'vertical', 'diagonal'.
+
+    Returns:
+        out_tensor (Tensor): Flipped feature map.
+    """
+    assert src_tensor.ndim == 4
+    valid_directions = ['horizontal', 'vertical', 'diagonal']
+    assert flip_direction in valid_directions
+    if flip_direction == 'horizontal':
+        out_tensor = torch.flip(src_tensor, [3])
+    elif flip_direction == 'vertical':
+        out_tensor = torch.flip(src_tensor, [2])
+    else:
+        out_tensor = torch.flip(src_tensor, [2, 3])
+    return out_tensor
+
+
+@DETECTORS.register_module()
+class CenterNet(CornerNet):
```

Comment:
I don't think it is necessary to inherit CornerNet.

Reply:
It's really unnecessary, I forgot to modify it.